### PR TITLE
data: Recommend team-maintained PPA for Ubuntu

### PR DESCRIPTION
**Pull request by smcv**
Alex's PPA can only be updated by him, which makes timely updates
difficult. To address this, we've created a `flatpak` team with `stable`
and `development` PPAs so that other interested developers (currently
only me, more volunteers welcome!) can also update packages.

### CODE REVIEW

**Comment by AsciiWolf** [comment]
Nice, great work! Now if only Ubuntu's GNOME Software fork would support Flatpaks...

**Comment by smcv** [comment]
> Now if only Ubuntu's GNOME Software fork would support Flatpaks...

That's up to Ubuntu developers, and I'm not one of those (except inasmuch as every upstream and Debian developer is indirectly helping Ubuntu).

If Ubuntu developers want to make Flatpak work better on Ubuntu, they're very welcome to either improve Ubuntu Software, help to maintain and test this PPA, or keep Flatpak and Flatpak-adjacent packages up-to-date in Ubuntu so that the PPA is unnecessary (I know @ahayzen already keeps an eye on Flatpak in older Ubuntu branches).

In a sense, the existence of this PPA is just a workaround for Ubuntu not having official Flatpak backports in `focal-backports` and `bionic-backports` (which would be the closest equivalent of what I'm doing in Debian).

When bionic reaches EOL, maybe we can stop recommending the PPA altogether? Or maybe it will need to stay because by the time bionic reaches EOL, the Flatpak 1.6.x in focal will be looking rather old?

**Comment by ahayzen** [comment]
If you install `gnome-software-plugin-flatpak` then you get support for Flatpak in Software, but if you use the snap of the Snap Store, then yeah you can't enable flatpak support yet (i guess that a flatpak interface would need to be added to snapd and then you'd need to convince them to bundle the plugin).

It probably makes sense for upstream to keep recommending the PPA, and yes a team PPA is much nicer but still comes with the  usual PPA downsides.

Backports would be nice, however when i investigated before I wasn't sure it was worth the effort due to multiple reasons
  * Multiple packages (ostree / flatpak) etc would need to be backported and need to have no diff when backporting
  * You need to backport through all supported releases to the release you are aiming for. So once 21.10 is out, if we wanted to backport a new flatpak from devel to 18.04, we would need to backport to 21.10, 21.04, 20.04, and 18.04
  * Even though the backports repository is enabled by default in recent Ubuntu versions it is no common for users to use it (I'm not sure how many users would know they can do `sudo apt install -t focal-backports flatpak` and would instead go searching for a PPA).  Eg look at how many packages have been backported for 20.04 https://packages.ubuntu.com/focal-backports/allpackages or 18.04 https://packages.ubuntu.com/bionic-backports/allpackages  Compare to Debian this is a very small amount of packages.

I have been trying to keep flatpak up to date in Ubuntu repos, but SRUs have become increasingly difficult and I suspect doing simple mciroreleases won't fly as easily as before and will need reasons for changes now.

So in conclusion, yes upstream recommending a team PPA is probably the best route.

**Review by ahayzen — APPROVED** [review]
LGTM :-)

**Comment by AsciiWolf** [comment]
Andrew, if you install `gnome-software-plugin-flatpak`, you will end up with two instances (snap and regular deb) of Software being installed and running at the same time. I have already tested it and it is far from ideal, it causes many problems and strange behavior. Anyway, this discussion is offtopic here.

**Comment by ahayzen** [comment]
> Andrew, if you install `gnome-software-plugin-flatpak`, you will end up with two instances (snap and regular deb) of Software being installed and running at the same time. I have already tested it and it is far from ideal, it causes many problems and strange behavior. Anyway, this discussion is offtopic here.

Yup I just end up removing the snap-store snap then install gnome-software-plugin-{flatpak,snap}, then you get the best of both worlds :-)

**Comment by AsciiWolf** [comment]
Merged, thanks. However, the actual live site won't be updated until #468 is fixed.
